src/Diagnostics.ModelsAndUtils/Models/Rendering.cs

```diff
@@ -346,10 +347,15 @@ public class TableColumnOption
         public int MaxWidth { get; set; }
 
         /// <summary>
-        /// Filter Selelction(Single, Multiple)
+        /// Filter Selection(Single, Multiple)
         /// </summary>
         public FileterSelectionOption SelectionOption { get; set; } = FileterSelectionOption.None;
 
+        /// <summary>
+        /// Default filter selection. Use '|' characters for multiple options (e.g. "option1 | option2")
+        /// </summary>
+        public string DefaultSelection { get; set; }
```

Comment:
Michi, do you think change `DefaultSelection` to an array of string will be better instead of using `|` to separate selection ?

Reply:
Agreed, it is better. Which do you prefer, string[] or List<string>?

Reply:
I think list is slightly better, so it can add default filter words more dynamic

Reply:
Change pushed. I used IEnumerable<string> like other classes.